net/mlvpn/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (C) 2017 Zhong Jianxin <azuwis@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=mlvpn
+PKG_VERSION:=2.3.1+20161231
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/zehome/MLVPN.git
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_VERSION:=14e17e80d6ebc50141c9e9548f12827c32dc92a0
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_SOURCE_VERSION).tar.xz
+PKG_MIRROR_HASH:=7928faa0bd4cb7dd5dc95fddd32772253bf3c48589b52cd0b5e80368c97aea75
+
+PKG_LICENSE:=BSD-2-Clause
+PKG_MAINTAINER:=Zhong Jianxin <azuwis@gmail.com>
+
+PKG_INSTALL:=1
+PKG_FIXUP:=autoreconf
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/mlvpn
+  TITLE:=Multi-Link Virtual Public Network
+  SECTION:=net
+  CATEGORY:=Network
+  URL:=https://zehome.github.io/MLVPN/
+  SUBMENU:=VPN
+  MENU:=1
+  DEPENDS:=+kmod-tun +libev +libpcap +libsodium
+endef
+
+define Package/mlvpn/description
+  Multi-Link Virtual Public Network
+  Bond your internet links to increase bandwidth (unlimited).
+  Secure your internet connection by actively monitoring your links and removing the faulty ones, without loosing your TCP connections.
+  Secure your internet connection to the aggregation server using strong cryptography.
+  Scriptable automation and monitoring.
+endef
+
+define Package/mlvpn/install
+	$(INSTALL_DIR) \
+		$(1)/usr/sbin \
+		$(1)/etc/init.d \
+		$(1)/etc/mlvpn \
+		$(1)/lib/upgrade/keep.d
+
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/mlvpn $(1)/usr/sbin/
+	install -m0700 files/mlvpn-updown $(1)/usr/sbin/
+	$(INSTALL_BIN) files/mlvpn.init $(1)/etc/init.d/mlvpn
+	$(INSTALL_DATA) files/mlvpn.upgrade $(1)/lib/upgrade/keep.d/mlvpn
```

Comment:
Add this instead manually adding the file to /lib/upgrade/keep.d/mlvpn please:
define Package/mlvpn/conffiles
/etc/mlvpn/
endef

It will have the same effect.